sdk/eventhub/Azure.Messaging.EventHubs/tests/Core/PartitionContextTests.cs

```diff
@@ -69,6 +69,7 @@ public void ReadLastEnqueuedEventPropertiesDelegatesToTheConsumer()
             Assert.That(information.Offset, Is.EqualTo(lastEvent.LastPartitionOffset), "The offset should match.");
             Assert.That(information.EnqueuedTime, Is.EqualTo(lastEvent.LastPartitionEnqueuedTime), "The last enqueue time should match.");
             Assert.That(information.LastReceivedTime, Is.EqualTo(lastEvent.LastPartitionPropertiesRetrievalTime), "The retrieval time should match.");
+            Assert.That(mockConsumer.IsClosed, Is.False, "The consumer should not have been closed or disposed of.");
```

Comment:
Why do we need to assert that? It doesn't feel like something we should worry about in this scenario.

Reply:
There was previously a difficult-to-detect issue where garbage collection was kicking in and the test was failing when the run was slow.   Adding this check here is mostly unnecessary other than it offers an active reference to the variable, preventing the compiler from optimizing such that it may be collected.   

I had originally left a comment with some silly does-nothing property access outside of an assert, but that looked and felt odd.